cluster-autoscaler/cloudprovider/gce/gce_price_model.go

```diff
@@ -283,9 +307,27 @@ func (model *GcePriceModel) NodePrice(node *apiv1.Node, startTime time.Time, end
 			price = price * preemptibleDiscount
 		}
 	}
-	// TODO: handle SSDs.
 
-	price += getAdditionalPrice(node.Status.Capacity, startTime, endTime)
```

Comment:
Do we want to keep the old logic in PodPrice()?

Reply:
I think so:
- It's much more difficult to tell what GPU a pod is requesting, since with NodeAffinity you can request any GPU from a given subset. What price should we assign in that case? 
- I don't think getting accurate pod price estimate is very important. If all expansion options have the same set of pods it doesn't matter at all. It's needed as a rough approximation for comparing options that help different set of pods, but even in this case its impact is likely limited in most cases (if some pods don't fit because a node is too small, preferred cluster shape part of expander formula is likely to determine result anyway). 
  - We already do something very similar for CPU - an e2 CPU and a c2 one have a very different price and pod is free to request whatever it wants via NodeAffinity.